internal/util/terraform.go

```diff
@@ -138,7 +161,7 @@ func (t *TerraformBinUtil) source() string {
 // Init will download the Terraform binary, put it into the .dce folder,
 // and then call init.
 func (t *TerraformBinUtil) Init(ctx context.Context, args []string) error {
-	logFile, err := t.FileSystem.OpenFileWriter(ctx.Value("deployLogFile").(string))
+	logFile, err := t.FileSystem.OpenFileWriter(ctx.Value(constants.DeployLogFile).(string))
```

Comment:
Re line 72: 

```
argv := []string{"init", "-no-color"}
argv = append(argv, args...)
```

So we're actually sneaking in a default value for `--tf-init-options` here.
Might be better to put this default in the Cobra config, so that users can override it.

Reply:
`-no-color` was there already, and I have no idea why and didn't ask. I just didn't want to go and remove it without knowing why. I'll ask around and see if I can find why it's there. I'll remove if there isn't a reason for it.

Reply:
`-no-color` is good if your running from a shell that can't process it (eg, from Jenkins or whatever) 
And actually a good default if you're writing to a file.